### PR TITLE
add phil-cough-data stack

### PR DESCRIPTION
**Pull request by philerooski**
Create S3 bucket for cough data data transfer.

### CODE REVIEW

**Comment by philerooski** [comment]
It's a lot of data and I assume it's related to a grant. We've tied specific buckets to specific grants in the past for expense tracking.

File: config/prod/phil-cough-data.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: cough-data
```
Comment (zaro0508) [inline config/prod/phil-cough-data.yaml:4]: stack_name and filename must match.

File: config/prod/phil-cough-data.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: phil-cough-data
```
Comment (zaro0508) [inline config/prod/phil-cough-data.yaml:4]: ohh wait, i think you said this bucket will likely live longer than you?  do you really want to have your name on the resources then?  AWS will append this `stack_name` to all the resources it creates.

Comment (philerooski) [reply config/prod/phil-cough-data.yaml:4]: Hahaha please just merge it 😂 This will be my legacy.

**Comment by philerooski** [comment]
@zaro0508 If you're still confused about why I opened the PR here rather than using scipool can you reach out to science LT and talk with them about what SOP is going forward with AWS resources that are tied to grants now that scipool and the service catalog are on the scene? I'm using what I believe to be SOP as of a few months ago but if it's changed in the meanwhile we should propagate that info to the rest of science.

**Pull request merged**